packages/ember-glimmer/lib/ember-metal-views/index.js

```diff
@@ -13,6 +15,17 @@ class DynamicScope {
   }
 }
 
+function setupRerenderLoop(renderer, view) {
+  // let LAST_TAG_VALUE = CURRENT_TAG.value();
+  run.backburner.on('begin', () => {
+    run.backburner.schedule('render', () => {
```

Comment:
So the Renderer's appendOutletView and appendTo methods would add a view to the list, and its remove method would remove that view? Then, when a view is removed, if the list is empty the begin handler is unregistered from the run loop?